mappings/net/minecraft/util/maths/Vec3d.mapping

```diff
@@ -0,0 +1,58 @@
+CLASS net/minecraft/class_26 net/minecraft/util/maths/Vec3d
+	FIELD field_1585 x D
+	FIELD field_1586 y D
+	FIELD field_1587 z D
+	FIELD field_1588 cache Ljava/util/List;
+	FIELD field_1589 count I
+	METHOD <init> (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD method_1292 resetCount ()V
+	METHOD method_1293 create (DDD)Lnet/minecraft/class_26;
+		ARG 0 x
+		ARG 2 y
+		ARG 4 z
+	METHOD method_1294 getDistance (Lnet/minecraft/class_26;)D
+		ARG 1 to
+	METHOD method_1295 interpolateWrtX (Lnet/minecraft/class_26;D)Lnet/minecraft/class_26;
```

Comment:
What does wrt mean

Reply:
with respect to

Reply:
here you give it a vec3d and an x coordinate and it generates a new vec3d with said x coordinate

Reply:
Weird code but ok